src/clusterfuzz/_internal/protos/uworker_msg.proto

```diff
@@ -170,7 +166,7 @@ message ProgressionTaskOutput{
   // When set to True, we delete the metadata related to: last_progression_min
   // and last_progression_max from the testcase on postprocess.
   optional bool clear_min_max_metadata = 9;
-  optional Json issue_metadata = 10;
+  map<string, string> issue_metadata = 10;
```

Comment:
Is the value always a string? And not great that we can have legal json ({1: 1}) that wont serialize here.